src/components/App.tsx

```diff
@@ -65,13 +65,21 @@ const App: FC<AppProps> = ({ demo }) => {
 
             setPatientRecords(records);
           });
+          client.patient?.read?.().then((resultPatient: fhir.Patient) => setPatient(resultPatient));
           setClient(client);
         });
     } else {
       setClient(new MockedFHIRClient());
-      setPatientRecords(demoRecords);
+      const url = config.get('demoPatientUrl') + demoId + '.json';
+      fetch(url)
+        .then(data => data.json())
+        .then(result => {
+          const resultPatient = result.find((r: DomainResource) => r.resourceType === 'Patient');
+          setPatientRecords(result);
```

Comment:
For future consideration only, not here, I wonder if it makes sense to combine the PatientRecordsProvider and PatientProvider

Reply:
Agreed, we didn't always have Context for both and when I decided to create the `PatientRecords` one I didn't think to combine them.